src/main/java/server/Server.java

```diff
@@ -0,0 +1,30 @@
+package main.java.server;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+
+public class Server {
+
+    private static final int PORT = 8080;
+
+    public static void main(String[] args) {
+        try {
+            ServerSocket serverSocket = new ServerSocket(PORT);
```

Comment:
![Bug](https://sonarsource.github.io/sonar-github/bug.png 'Bug') Bug: Use try-with-resources or close this "ServerSocket" in a "finally" clause. ([squid:S2095](https://sonarcloud.io/organizations/jangchulwoon-github/rules#rule_key=squid%3AS2095))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=groupId%3AHTTPServer&pullRequest=1&issues=AWQTU_q4bXXTsx7O43dQ&open=AWQTU_q4bXXTsx7O43dQ)

[ISSUE_KEY]: AWQTU_q4bXXTsx7O43dQ